lib/dl_api_lib/dl_api_lib/connector_availability/base.py

```diff
@@ -223,6 +285,24 @@ class ConnectorAvailabilityConfig(SettingsBase):
         env_var_converter=conn_type_set_env_var_converter,
         missing_factory=set,
     )
+    icon_src: ConnectorIconSrcConfig = attr.ib(default=None)
+
+    _icon_by_conn_type: dict[str, dict[str, Any]] = attr.ib(factory=dict)
+
+    def fill_icon_dict_by_conn_type(self) -> None:
+        if not self._icon_by_conn_type and self.icon_src is not None:
```

Comment:
to avoid an extra level of indentation, you can invert the statement and return from it:
```
if self._icon_by_conn_type or self.icon_src is None:
    return
```